realm/src/main/java/io/realm/Realm.java

```diff
@@ -1018,6 +1230,28 @@ public void writeEncryptedCopyTo(File destination, byte[] key) throws IOExceptio
         return realmObjects;
     }
 
+    /**
+     * Updates a list of existing RealmObjects that is identified by their {@link io.realm.annotations.PrimaryKey} or create a
+     * new copy if no existing object could be found.
+     *
+     * @param objects   List of objects to update or copy into Realm.
+     * @return A list of all the new or updated RealmObjects.
+     *
+     * @throws java.lang.IllegalArgumentException if RealmObject is {@code null} or doesn't have a Primary key defined.
+     * @see {@link #copyToRealm(Iterable)}
+     */
+    public <E extends RealmObject> List<E> copyToRealmOrUpdate(Iterable<E> objects) {
```

Comment:
We should yes, but it is the same as for copyToRealm(Iterable). We need to be able to return a tableview containing all inserted values which currently isn't feasible, at least that was how I remember the conclusion from when we discussed it.